README.md

```diff
@@ -10,9 +10,10 @@ Goals:
 
 ## Installation notes
 
-This assumes you are on MacOS with HomeBrew installed. `make install` will brew install `gitleaks`. The install will:
+`make install` will install `gitleaks`. The install will:
 
 * install `gitleaks`
+  - **Note:** Only `gitleaks` version 8 is currently supported.
```

Comment:
Putting this in here just in case anyone does decide to install `gitleaks` via Homebrew instead of using `make install`